src/test/java/com/google/api/codegen/testdata/java_test_library.baseline

```diff
@@ -626,7 +628,7 @@ public class LibraryServiceTest {
     expectedResponses.add(expectedResponse);
     mockLibraryService.setResponses(expectedResponses);
 
-    String formattedName = LibraryServiceApi.formatBookName("[SHELF]", "[BOOK]");
+    BookName formattedName = BookName.create("[SHELF]", "[BOOK]");
```

Comment:
It occurs to me that we could drop the addition of 'formatted' to parameter names since we have the resource types now